### DISCUSSION TITLE
A different payment way

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#478] [Q&A]
Instead of online payment I would like to use a coupon system. For example, a half-day event would cost one coupon and a one-day event would cost 2 coupons. Beyond one day, you simply multiply by the number of days to find out the cost in coupons. The card including the number of coupons would be implemented with a membership. The price of the membership would depend on the number of coupons purchased. For example: 1 coupon = €15, 2 coupons = €26 (a coupon costs €13), 20 coupons = €200 (a coupon costs €10). As many memberships as purchasing systems.
To book an event, you need to have coupons, otherwise you'll be directed to a payment gateway to buy them. Membership is used to memorize purchases, usage when booking, history...
Does this system seem viable to you? Is there another way, for example with the discount system? Has anyone already developed this type of payment?

**Comment by liedekef** [comment]
Sorry, not possible with EME. Maybe you can achieve something like this with discounts of type "Code" but the coupon-thing itself is beyond EME.
